address/mock.go

```diff
@@ -14,8 +15,20 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// RandProofCourierAddr returns a proof courier address with fields populated
+// with valid but random values.
+func RandProofCourierAddr(t testing.TB) *url.URL {
```

Comment:
non-blocking nit: returns a static address? Could be useful to have a rationale or TODO here if we want it to be random later on.